PS4History.py

```diff
@@ -166,6 +167,39 @@ def allow_user(u):
 
         return stats # { mode: { user: { [stat]: int ... }, ... } }
 
+
+
+    def summary_elo(self, channel, name = None, year = None):
+
+        ## TODO handle non fifa stuff here
+        ## or hardcode to fifa2 channel
```

Comment:
need @bobrippling  to fix this part to work for all game types plz

Reply:
Sortid :+1: